modules/custom/social_demo/content/entity/group.yml

```diff
@@ -7,7 +7,12 @@
   uid: 44fc3642-da2f-11e5-b5d2-0a1d41d68578
   created: -60 day|15:30
   label: Official Group Educational Assistance Association
-  type: open_group
+  type: flexible_group
+  field_flexible_group_visibility: community
```

Comment:
Seems like the `field_flexible_group_visibility` isn't taken in to account during creation for any of the groups.
![Screenshot 2024-01-25 at 08 38 57](https://github.com/goalgorilla/open_social/assets/16667281/e10c590c-aa15-4315-aec2-2d8b32012bf4)
![Screenshot 2024-01-25 at 08 47 24](https://github.com/goalgorilla/open_social/assets/16667281/cc499580-0b46-479c-9782-c2261e78a85e)
![Screenshot 2024-01-25 at 08 47 46](https://github.com/goalgorilla/open_social/assets/16667281/71f3d1d9-b068-44c4-9930-98fc52bfb3fe)